spec/models/spree/order_spec.rb

```diff
@@ -1141,12 +1141,9 @@
       end
 
       context "when finalized fee adjustments exist on the order" do
-        let(:payment_fee_adjustment) { order.all_adjustments.payment_fee.first }
-        let(:shipping_fee_adjustment) { order.shipment_adjustments.first }
-
         before do
-          payment_fee_adjustment.finalize!
-          shipping_fee_adjustment.finalize!
+          order.all_adjustments.payment_fee.each(&:finalize!)
+          order.shipment_adjustments.each(&:finalize!)
```

Comment:
We could maybe just do `order.all_adjustments.each(&:finalize!)` here for simplicity. Not a big deal though.